frontend-react/src/styles/_typography.scss

```diff
@@ -1,6 +1,17 @@
 // Type styles for markdown-driven pages & any overrides for USWDS typography
 
 // Headings
+// FUTURE_TODO: Decipher the ancient texts on how to apply these directly to USWDS token definitions.
+h4 {
```

Comment:
Did we want to add the weight here? per Figma

Reply:
The existing USWDS h4 style rules apply bold weight already.